src/server/req_runjob.c

```diff
@@ -1377,19 +1377,29 @@ post_sendmom(struct work_task *pwt)
 				        jobp->ji_qs.ji_destin, pbs_errno, reject_msg);
 			else
 				sprintf(log_buffer,
-				        "send of job to %s failed error = %d",
-				        jobp->ji_qs.ji_destin, pbs_errno);
+					"send of job to %s failed error = %d",
+					jobp->ji_qs.ji_destin, pbs_errno);
+
 			log_event(PBSEVENT_JOB, PBS_EVENTCLASS_JOB, LOG_INFO, jobp->ji_qs.ji_jobid, log_buffer);
 			snprintf(log_buffer, LOG_BUF_SIZE,
 					"Not Running: PBS Error: %s", pbse_to_txt(PBSE_MOMREJECT));
-			if (jobp->ji_qs.ji_svrflags & JOB_SVFLG_SubJob)
-				job_attr_def[(int) JOB_ATR_Comment].at_decode(
+
+			/* job array case */
```

Comment:
Let's have this comment be a little more helpful.  How about saying that the comment is set to a begun message when it starts and doesn't get changed after that.  Just saying it is the job array case is a little confusing.